deploy/kubernetes/overlays/stable/ecr/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../../base
+images:
+  - name: amazon/aws-efs-csi-driver
+    newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-efs-csi-driver
+    newTag: v0.3.0
+  - name: quay.io/k8scsi/livenessprobe
+    newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/csi-liveness-probe
+    newTag: v1.1.0
+  - name: quay.io/k8scsi/csi-node-driver-registrar
+    newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/csi-node-driver-registrar
+    newTag: v1.1.0
```

Comment:
should this be different PR? (I don't particularly mind including this, just want to make sure you didn't commit stuff by accident)

Reply:
Yeah that was intentionally. I agree that should be at least a separate commit.